vmware_aria_operations_integration_sdk/mp_init.py

```diff
@@ -357,6 +377,7 @@ def main() -> None:
                 icon_file,
                 language,
                 template_style,
+                package_name,
```

Comment:
I don't think we want to keep adding language-specific parameters here. Perhaps we could have a class representing each language that can prompt for and store any language-specific values it needs?

Reply:
Also, I don't see `package_name` being set if the language is not Java -- which I think will error.